### ISSUE TITLE
[Tuning] Zscaler.ZIA.PasswordExpiration

### ISSUE THREAD

**Issue by zestysoft** [panther-labs/panther-analysis#1876]
## Rule ID

Zscaler.ZIA.PasswordExpiration

## Description

This rule has a bug that causes false positive alerts when both `passwordExpirationEnabled` fields are missing from an event.

The rule uses different default values for `password_exp_pre` and `password_exp_post`:

```python
password_exp_pre = event.deep_get(
    "event",
    "preaction",
    "passwordExpirationEnabled",
    default="<PRE_PASSWORD_EXPIRATION_NOT_FOUND>",
)
password_exp_post = event.deep_get(
    "event",
    "postaction",
    "passwordExpirationEnabled",
    default="<POST_PASSWORD_EXPIRATION_NOT_FOUND>",
)
if action == "UPDATE" and category == "LOGIN" and password_exp_pre != password_exp_post:
    return True
```

When both `passwordExpirationEnabled` fields are missing from an event:
- `password_exp_pre` = `"<PRE_PASSWORD_EXPIRATION_NOT_FOUND>"`
- `password_exp_post` = `"<POST_PASSWORD_EXPIRATION_NOT_FOUND>"`

Since these two strings are different, the inequality check `password_exp_pre != password_exp_post` evaluates to `True`, causing false positive alerts for any UPDATE event in the LOGIN category that lacks these fields.
